webapp/components/interop.js

```diff
@@ -429,182 +502,228 @@ class InteropDashboard extends PolymerElement {
+                  <optgroup label="[[section.name]]">
+                    <template is="dom-repeat" items={{section.rows}} as="focusArea">
+                      <option value$="[[focusArea]]" selected="[[isSelected(focusArea)]]">
+                        [[getRowInfo(focusArea, 'description')]]
+                      </option>
+                    </template>
+                  </optgroup>
+                </template>
+              </select>
+            </div>
 
-      <section class="focus-area-section">
-        <h2 class="focus-area-header">Scores over time</h2>
-
-        <p class="prose">
-          Here you can see how focus areas are improving over time.
-          The more tests that pass, the higher the score.
-        </p>
-
-        <div class="focus-area">
-          <select id="featureSelect">
-            <option value="summary">Summary</option>
-            <template is="dom-repeat" items="{{getYearProp('tableSections')}}" as="section" filter="{{filterGroupSections()}}">
-              <optgroup label="[[section.name]]">
-                <template is="dom-repeat" items={{section.rows}} as="focusArea">
-                  <option value$="[[focusArea]]" selected="[[isSelected(focusArea)]]">
-                    [[getRowInfo(focusArea, 'description')]]
-                  </option>
+            <div id="featureReferenceList">
+              <template is="dom-repeat" items="[[featureLinks(feature)]]">
+                <template is="dom-if" if="[[item.href]]">
+                  <a href$="[[item.href]]">[[item.text]]</a>
                 </template>
-              </optgroup>
-            </template>
-          </select>
-        </div>
+                <template is="dom-if" if="[[!item.href]]">
+                  <span>[[item.text]]</span>
+                </template>
+              </template>
+            </div>
 
-        <div id="featureReferenceList">
-          <template is="dom-repeat" items="[[featureLinks(feature)]]">
-            <template is="dom-if" if="[[item.href]]">
-              <a href$="[[item.href]]">[[item.text]]</a>
-            </template>
-            <template is="dom-if" if="[[!item.href]]">
-              <span>[[item.text]]</span>
-            </template>
-          </template>
+            <interop-feature-chart year="[[year]]"
+                                  data-manager="[[dataManager]]"
+                                  stable="[[stable]]"
+                                  feature="{{feature}}">
+            </interop-feature-chart>
+          </section>
         </div>
-
-        <interop-feature-chart year="[[year]]"
-                               data-manager="[[dataManager]]"
-                               stable="[[stable]]"
-                               feature="{{feature}}">
-        </interop-feature-chart>
-      </section>
+      </div>
       <footer class="compat-footer">
```

Comment:
Most of the footer text here has been moved into the the grid's description, and the footer now only displays links to specific years.